src/screens/reader/utils/sanitizeChapterText.ts

```diff
@@ -31,7 +31,11 @@ export const sanitizeChapterText = (
   });
   if (text) {
     if (options?.removeExtraParagraphSpacing) {
-      text = text.replace(/<\s*br[^>]*>/gi, '\n').replace(/\n{2,}/g, '\n\n');
+      text = text
+        .replace(/[\u200B-\u200D\uFEFF]/g, '')
+        .replace(/(?:<br>\s*<\/?br>)+/g, '')
+        .replace(/<br>\s*(?=<\/?p>)/g, '')
+        .replace(/\n\n+/g, '\n');
```

Comment:
```suggestion
      text = text
        .replace(/[\u200B-\u200D\uFEFF]+/g, '')
        .replace(/(<br>\s*<\/?br>)+/g, '')
        .replace(/<br>\s*(?=<\/?p>)/g, '')
        .replace(/\n{2,}/g, '\n');
```

Reply:
-``\n\n+`` is faster than ``\n{2,}``
- the ``?:`` helps the regex performance. Idk why. The more complex is a group the more evident it is.